ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicaThread.java

```diff
@@ -58,6 +65,17 @@ public class VcrReplicaThread extends ReplicaThread {
   protected AzureCloudConfig azureCloudConfig;
   protected VerifiableProperties properties;
   protected CloudDestination cloudDestination;
+  protected HashMap<String, List<Pair<RemoteReplicaInfo, MessageInfo>>> blobToReplicas;
```

Comment:
It seems there is no code cleaning up these two maps so these two maps would probably blow up the entire memory.